### ISSUE TITLE
CAN-FD not implemented?

### ISSUE THREAD

**Issue by Oremuss** [Kvaser/pycanlib#3]
Hi guys. I am not able to run pycanlib with CAN FD. 

I downloaded example "canfd_sample.py" but cant run it because of error: canlib.canlib.exceptions.CanNotImplementedError: Not implemented (-32).

I am using Windows 10 (64 bit) and using Kvaser Leaf Pro HS v2. I downloaded latest kvaser_drivers_setup.exe and canlib.exe from kvaser official website. I am using python 3.9 and installed latest canlib. 
Here I upload error log:

Traceback (most recent call last):
  File "d:\users\pythonDir\canfd.py", line 26, in <module>
    ch0.set_bus_params_tq(params_arbitration, params_data)
  File "C:\Python39\lib\site-packages\canlib\canlib\channel.py", line 589, in set_bus_params_tq
    dll.canSetBusParamsFdTq(
  File "C:\Python39\lib\site-packages\canlib\canlib\dll.py", line 188, in _error_check
    raise can_error(result)
canlib.canlib.exceptions.CanNotImplementedError: Not implemented (-32)

**Comment by TroyVia** [comment]
What version of firmware is in the Kvaser Leaf Pro HS v2 unit you are using?

Make sure you are using the latest firmware in the unit by downloading the latest version of the Kvaser Frimware Update Tool here:  https://www.kvaser.com/download/?utm_source=software&utm_ean=7330130981614&utm_status=latest.  Unpack the provided file into a temporary folder and run the Update.exe.

Also, are you using version 1.16 or greater of PyCANLib?

**Comment by Oremuss** [comment]
That worked! :) Thank you for your help.
I was testing CANFD sending/receiveing with two Kvasers  (Leaf Pro HS v2) using Kvaser CanKing SW and it was sending and receiving properly. After your suggestion I checked firmware version of both my Kvasers and both were out of date with version (3.12) now its 3.28 (and from release notes I found out that CANFD support was added in 3.14). Thou it is weird that CANFD communication worked with CanKing SW without firmware update.
